src/github/api/mod.rs

```diff
@@ -23,6 +23,12 @@ type GHRepositoryState = RepositoryState<GithubRepositoryClient>;
 
 type RepositoryMap = HashMap<GithubRepoName, GHRepositoryState>;
 
+fn base_github_html_url() -> Url {
+    let url = std::cell::OnceCell::new();
```

Comment:
Now that I see this, this actually creates the `OnceCell` everytime the function is called. It would need to be `static`, but then it would need to implement `Sync`, so it needs to be wrapped in a thread local or it would need to be `OnceLock` :laughing:

Yeah, my code below is totally wrong =D

Let's just returns `&'static str` here, we don't need the `Url` for this specific URL, I think.

Reply:
Ah, that's explain why. I didn't understand the OnceCell part, so I just copy it. Do you want me to do the same for the other function too?